engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/BpmnEventTypeTest.java

```diff
@@ -322,6 +322,88 @@ void executeInstance() {
+                  .startEvent()
+                  .intermediateThrowEvent("throw", i -> i.escalation(escalationCode()))
+                  .endEvent()
+                  .done();
+            }
+
+            @Override
+            void executeInstance() {
+              super.executeInstance(Collections.emptyMap());
+            }
+          },
+          new BpmnEventTypeScenario(
+              "Escalation Throw Event",
+              BpmnElementType.INTERMEDIATE_THROW_EVENT,
+              BpmnEventType.ESCALATION) {
+            @Override
+            BpmnModelInstance modelInstance() {
+              return Bpmn.createExecutableProcess(processId())
+                  .eventSubProcess(
+                      "subprocess",
+                      sp -> sp.startEvent("catch", s -> s.escalation(escalationCode()).endEvent()))
+                  .startEvent()
+                  .intermediateThrowEvent(elementId(), e -> e.escalation(escalationCode()))
+                  .endEvent()
+                  .done();
+            }
+
+            @Override
+            void executeInstance() {
+              super.executeInstance(Collections.emptyMap());
+            }
+          },
+          new BpmnEventTypeScenario(
+              "Escalation Catch Event", BpmnElementType.BOUNDARY_EVENT, BpmnEventType.ESCALATION) {
+            @Override
+            BpmnModelInstance modelInstance() {
+              return Bpmn.createExecutableProcess(processId())
+                  .startEvent()
+                  .subProcess(
+                      "subprocess",
+                      sp ->
+                          sp.embeddedSubProcess()
+                              .startEvent()
+                              .intermediateThrowEvent("throw", e -> e.escalation(escalationCode()))
+                              .endEvent())
+                  .boundaryEvent(elementId(), b -> b.escalation(escalationCode()))
+                  .cancelActivity(false)
+                  .endEvent()
+                  .done();
+            }
+
+            @Override
+            void executeInstance() {
+              super.executeInstance(Collections.emptyMap());
+            }
+          },
+          new BpmnEventTypeScenario(
+              "Escalation End Event", BpmnElementType.END_EVENT, BpmnEventType.ESCALATION) {
+            @Override
+            BpmnModelInstance modelInstance() {
+              return Bpmn.createExecutableProcess(processId())
+                  .eventSubProcess(
+                      "subprocess",
+                      sp -> sp.startEvent("catch", s -> s.escalation(escalationCode()).endEvent()))
+                  .startEvent()
+                  .endEvent(elementId(), e -> e.escalation(escalationCode()))
+                  .done();
+            }
+
+            @Override
+            void executeInstance() {
+              super.executeInstance(Collections.emptyMap());
+            }
```

Comment:
❌ There is no need to override this method. The default behavior already starts an instance without variables.